### DISCUSSION TITLE
VScode pylance functionality in qmd files

### DISCUSSION THREAD

**Discussion by johnros** [quarto-dev/quarto-cli#4173] [Q&A]
Hello all
Is there a way to edit .qmd files in vscode, while enabeling all the fnuctionality of VScode/pylance? 
Thinks like going to definiton, references, renaming, refactoring, linting, ...

**Comment by ntluong95** [comment]
Is there any news about this?

**Comment by sbalint98** [comment]
+1

**Reply by mcanouil** [reply]
@sbalint98 please use reactions instead of such comment which does not allow to account for your interest and also triggers unnecessary notifications.
